.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: release
+name: GitHub Release
```

Comment:
I changed this name @KtorZ to differentiate more accurately what the two workflows were doing.